packages/compartment-mapper/src/node-modules.js

```diff
@@ -49,7 +49,9 @@ const q = JSON.stringify;
  * @param {string} abs - a fully qualified URL
  * @returns {string}
  */
-const resolveLocation = (rel, abs) => new URL(rel, abs).toString();
+const resolveLocation = (rel, abs) => {
+  return new URL(rel, abs).toString();
+};
```

Comment:
Curious, any reason for the explicit block & `return` change?

Reply:
Incidental drift. I had a console log in there for debugging. I’m inclined to leave it this way.